### ISSUE TITLE
Keyboard shortcut conflict (CTRL+SHIFT+V)

### ISSUE THREAD

**Issue by defaultsecurity** [linkwarden/browser-extension#80]
**Describe the bug**
The browser extension's keyboard shortcut CTRL+SHIFT+V conflicts with the commonly used shortcut "paste plain text".

**To Reproduce**
When I'm focused in any textbox and press CTRL+SHIFT+V, the text is not pasted, instead the extension's popup appears.

**Expected behavior**
I would expect the text to be pasted as usual.

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Brave
 - Version: Brave 1.168.137, Linkwarden Extension 1.2.6

**Solution**
Change the shortcut or make it customizeable.

**Comment by mhsayan** [comment]
Yes, this one is very much annoying. As this shortcut is used for quick operations but you can create a bookmarklet like [linkding](https://github.com/sissbruecker/linkding) to solve this issue.

**Comment by crkos** [comment]
Hello this is changed in the dev version, It uses Ctr + Shift + F , i'm still working on a feature that will solve all of that

**Comment by dankerthrone** [comment]
yes please do something about this. It annoys the hell out of me, as I use a lot of web-based authoring tools and copy-paste a lot between them. Now everytime I try to get my work done, I instead save a bookmark.
